spec/units/derivatives_spec.rb

```diff
@@ -30,4 +30,30 @@ class CustomFile < ActiveFedora::Base
       end
     end
   end
+
+  [:ffmpeg_path, :libreoffice_path, :temp_file_base, :fits_path, :kdu_compress_path,
+   :kdu_compress_recipes, :enable_ffmpeg, :source_file_service, :output_file_service].each do |method|
+    describe method.to_s do
+      it 'returns the config value' do
+        expect(subject.send(method)).to eq subject.config.send(method)
+      end
+    end
+    describe "#{method}=" do
+      it 'stores config changes' do
+        expect { subject.send("#{method}=", "new_value") }.to change { subject.config.send(method) }.from(subject.config.send(method)).to("new_value")
+      end
+    end
+  end
+
+  describe 'reset_config!' do
+    it "lets you reset the configuration" do
```

Comment:
Reading the specs, I feel that `it "resets the configuration" do` is more assertive and possibly descriptive than the current language.